packages/gen-typescript-declarations/scripts/update-goldens.js

```diff
@@ -30,10 +30,18 @@ const {generateDeclarations} = require('../lib/gen-ts');
 const fixturesDir = path.join(__dirname, '..', 'src', 'test', 'fixtures');
 const goldensDir = path.join(__dirname, '..', 'src', 'test', 'goldens');
 
-fsExtra.emptyDirSync(goldensDir);
+const filter = process.env.UPDATE_GOLDENS_FILTER || '';
```

Comment:
Why add the filter?

Reply:
It's to make it easy to work on one of the test cases at a time with a faster development loop, since it takes a few seconds per case.